Cabal-syntax/src/Distribution/Compat/CharParsing.hs

```diff
@@ -15,28 +19,27 @@
 -- Parsers for character streams
 --
 -- Originally in @parsers@ package.
---
------------------------------------------------------------------------------
 module Distribution.Compat.CharParsing
-  (
-  -- * Combinators
-    oneOf        -- :: CharParsing m => [Char] -> m Char
-  , noneOf       -- :: CharParsing m => [Char] -> m Char
-  , spaces       -- :: CharParsing m => m ()
-  , space        -- :: CharParsing m => m Char
-  , newline      -- :: CharParsing m => m Char
-  , tab          -- :: CharParsing m => m Char
-  , upper        -- :: CharParsing m => m Char
-  , lower        -- :: CharParsing m => m Char
-  , alphaNum     -- :: CharParsing m => m Char
-  , letter       -- :: CharParsing m => m Char
-  , digit        -- :: CharParsing m => m Char
-  , hexDigit     -- :: CharParsing m => m Char
-  , octDigit     -- :: CharParsing m => m Char
+  ( -- * Combinators
+    oneOf -- :: CharParsing m => [Char] -> m Char
```

Comment:
Another example where it's easier to read when vertically aligned. I guess, this kind of thing is what Andreas had alluded to.

Reply:
Yes it's always a matter of tradeoff: Are you privileging vertical readability or minimising diffs in PRs?

Reply:
I think the former (readability). But if there's a vast support for the latter, i will happily take it.

Reply:
The diffs in LANGUAGE pragmas and export lists never bothered me tbh. In the code they're occasionally annoying but not very often either.

Reply:
Remember that you can locally turn off (f)ormolu if you want to preserve alignment in some places.